tests/Http/Productions/ProductionsWriteControllerTest.php

```diff
@@ -101,6 +102,22 @@ function (AddEventToProduction $command) use ($productionId, $eventId) {
         $this->controller->addEventToProduction($productionId->toNative(), $eventId);
     }
 
+    /**
+     * @test
+     */
+    public function it_can_remove_an_event_from_an_existing_production(): void
+    {
+        $productionId = ProductionId::generate();
+        $eventId = Uuid::uuid4();
+        $this->commandBus->expects($this->once())->method('dispatch')->willReturnCallback(
```

Comment:
Btw Broadway has a `TraceableCommandBus` for testing purposes. You have to set it up as:
```
$this->commandBus = new Broadway\CommandHandling\Testing\TraceableCommandBus();
$this->commandBus->record();
```
And then in your test you can just do `$this->commandBus->getRecordedCommands()`.

Reply:
My bad for not mentioning this in earlier PRs 😉

Reply:
That's probably going to make the tests a lot more readable. Will change it in the follow-up PR.